app/src/main/java/io/intrepid/contest/utils/SpannableUtil.java

```diff
@@ -0,0 +1,41 @@
+package io.intrepid.contest.utils;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.support.annotation.ColorRes;
+import android.support.annotation.NonNull;
+import android.support.v4.content.ContextCompat;
+import android.text.SpannableString;
+import android.text.Spanned;
+import android.text.style.ForegroundColorSpan;
+import android.text.style.StyleSpan;
+
+public class SpannableUtil {
```

Comment:
Since you created this you can use it in `InvitationIntroFragment.showPermissionDeniedMessage()`, otherwise it's a bit of a wasted effort just for one use